README.Rmd

```diff
@@ -0,0 +1,62 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# Pandora
+
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/Pandora-IsoMemo/pandora-data/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/Pandora-IsoMemo/pandora-data/actions/workflows/R-CMD-check.yaml)
+<!-- badges: end -->
+
+This R-Package is a wrapper for the [CKAN API](https://docs.ckan.org/en/2.9/api/) 
+of the Pandora data platform.
+
+[Pandora](https://pandora.earth/) seeks a better understanding of the human 
+story by offering technological solutions to make historical and archaeological 
+data more accessible and easily findable. It is a grassroots initiative where 
+research communities from across various academic disciplines self-curate 
+datasets and self-manage community  membership and roles. These datasets may be 
```

Comment:
Please just remove the part:

> These datasets may be deposited externally or
within the Pandora data platform itself. Datasets can be assigned DOIs
and are indexed so that they are findable using scientific search
engines.

A reader might think that this could be part of our R package